CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.14.7] - 2018-12-27
+
+### Improved
+
+- Tracing shows more relevant data
+- Middleware is provided to improve tracing
```

Comment:
How does it improve tracing?

* If it is worth explaining it in the changelog, please do it.
* If it is not, please remove this from the changelog.